src/History/components/MyLecueBook/index.tsx

```diff
@@ -21,13 +21,12 @@ function MyLecueBook(props: LecueBookProps) {
   } = props;
   const [noteCount, setNoteCount] = useState('');
   const [modalOn, setModalOn] = useState(false);
-  const [favorite, setFavorite] = useState(isFavorite);
 
   const navigate = useNavigate();
 
   const deleteMutation = useDeleteMyBook();
   const FavoritePostMutation = usePostFavorite();
-  const FavoriteDeleteMutation = useDeleteFavorite('mypage');
+  const FavoriteDeleteMutation = useDeleteFavorite('myLecueBook');
 
   const convertNoteCount = (noteNum: number) => {
     setNoteCount(noteNum.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ','));
```

Comment:
ASK: 이 부분은 위해 noteCount를 number 타입으로 선언한다면 아래와 같이 정의해도 잘 동작하는 것 같은데,
정규식을 사용해서 변환해준 이유가 따로 있을까요??
```suggestion
    setNoteCount(noteNum);
```

Reply:
++ ASK:  해당 정규식이 어떤 걸 의미하는지도 궁금합니다 !
콘솔 찍어봤을 때 num -> string으로 변환되는 것 말구 큰 차이를 모르겠더라구요..!